tests/moduletemplates/moduletemplate_template_operator_regular_new_ocm.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: operator.kyma-project.io/v1beta2
+kind: ModuleTemplate
+metadata:
+  name: template-operator-regular
+  namespace: kcp-system
+  labels:
+    "operator.kyma-project.io/module-name": "template-operator"
+  annotations:
+    "operator.kyma-project.io/is-cluster-scoped": "false"
+    "operator.kyma-project.io/module-version": "1.0.0-new-ocm-format"
+spec:
+  channel: regular
+  mandatory: false
+  descriptor:
+    component:
+      componentReferences: []
+      creationTime: "2024-07-02T09:16:42Z"
+      name: kyma-project.io/module/template-operator
+      provider: kyma-project.io
+      repositoryContexts:
+        - baseUrl: europe-west3-docker.pkg.dev
+          componentNameMapping: urlPath
+          subPath: sap-kyma-jellyfish-dev/template-operator
+          type: OCIRegistry
+      resources:
+        - access:
+            imageReference: europe-docker.pkg.dev/kyma-project/prod/template-operator:1.0.0
+            type: ociArtifact
+          digest:
+            hashAlgorithm: SHA-256
+            normalisationAlgorithm: ociArtifactDigest/v1
+            value: 03a194e1dca2421755cec5ec1e946de744407e6e1ca3b671f715fee939e8d1fb
+          name: module-image
+          relation: external
+          type: ociArtifact
+          version: 1.0.0
+        - access:
+            localReference: sha256:d2cc278224a71384b04963a83e784da311a268a2b3fa8732bc31e70ca0c5bc52
+            mediaType: application/x-tar
+            type: localBlob
+          digest:
+            hashAlgorithm: SHA-256
+            normalisationAlgorithm: genericBlobDigest/v1
+            value: d2cc278224a71384b04963a83e784da311a268a2b3fa8732bc31e70ca0c5bc52
+          name: raw-manifest
+          relation: local
+          type: directory
+          version: 1.0.0
+        - access:
+            localReference: sha256:9230471fa6a62ff7b1549e8d0e9ccb545896fabadf82d2ec4503fc798d2bcd8a
+            mediaType: application/x-tar
+            type: localBlob
+          digest:
+            hashAlgorithm: SHA-256
+            normalisationAlgorithm: genericBlobDigest/v1
+            value: 9230471fa6a62ff7b1549e8d0e9ccb545896fabadf82d2ec4503fc798d2bcd8a
+          name: default-cr
+          relation: local
+          type: directory
+          version: 1.0.0
+        - access:
+            localReference: sha256:89256dd62ebab4f7f02482084b91301087a084426752b00754e14afd3575e019
+            mediaType: application/octet-stream
+            type: localBlob
+          digest:
+            hashAlgorithm: SHA-256
+            normalisationAlgorithm: genericBlobDigest/v1
+            value: 89256dd62ebab4f7f02482084b91301087a084426752b00754e14afd3575e019
+          name: managed-resources
```

Comment:
Needs to be renamed now to `associated-resources` as discussed this morning? Decision record has also been updated accordingly: https://github.com/kyma-project/lifecycle-manager/issues/1587